documentation/Demo-Server-Guide.md

```diff
@@ -159,86 +159,70 @@ Logs are in `/var/log/nginx`
 
 ## docker-compose
 
-docker-compose will complain if directories are not present, create them if needed (they will persist when restarting image, even if `docker-compose down` was run) - use `docker compose down` alternativelly.
```

Comment:
> You removed the part where I'm expliciting that we need to run using `docker compose`

Where is this? That's not what the highlighted bit here is about, this is just removing  information that is no longer correct or relevant:

> docker-compose will complain if directories are not present, create them if needed

Reply:
Just the "- use `docker compose down` alternativelly." part

Reply:
Which is actually not required here, but for the `docker-compose up` it would actually be. `docker compose up`

Reply:
> Just the "- use `docker compose down` alternativelly." part

That's further down. This section is divided into:

- What docker compose does 
- Commands for launching
- Commands for stopping (i.e. docker-compose down)

> but for the docker-compose up it would actually be. docker compose up
This is a bit confusing -- looks like at the moment, different commands are required on different systems: https://stackoverflow.com/a/66516826

Probably best to just leave it with the one that works on our demo server (docker-compose), and add a note somewhere that docker-compose might need to be replaced with docker compose, would you say?

Reply:
Yes, indeed. I'll add a line with that in here. I'm planning to upgrade the server now. Would be ok for me to merge this one in?

Reply:
Yes, but you'll need to do some fiddling with the folders on the server when you first load it up. Do you want me to talk you through that?